repair_picking_after_done/wizards/repair_move_transfer.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+from odoo.tools.float_utils import float_compare
+
+
+class MrpInventoryProcure(models.TransientModel):
+    _name = "repair.move.transfer"
+    _description = "Create an internal transfer from repaired moves"
+
+    repair_order_id = fields.Many2one(
+        "repair.order", string="Repair Order", required=True
+    )
+    location_dest_id = fields.Many2one(
+        "stock.location", string="Destination location", required=True
+    )
+    quantity = fields.Float("Quantity to transfer", required=True)
+    remaining_quantity = fields.Float("Remaining Quantity to Transfer")
+
+    def _get_picking_type(self):
+        self.ensure_one()
+        warehouse = self.repair_order_id.location_id.warehouse_id
+        return warehouse.int_type_id
+
+    def _prepare_picking_vals(self):
+        return {
+            "partner_id": False,
+            "user_id": False,
+            "picking_type_id": self._get_picking_type().id,
+            "move_type": "direct",
+            "location_id": self.repair_order_id.location_id.id,
+            "location_dest_id": self.location_dest_id.id,
+        }
+
+    def _prepare_stock_move_vals(self, picking):
+        self.ensure_one()
+        return {
+            "name": self.repair_order_id.product_id.name,
+            "product_id": self.repair_order_id.product_id.id,
+            "location_id": self.repair_order_id.location_id.id,
+            "location_dest_id": self.location_dest_id.id,
+            "picking_id": picking.id,
+            "state": "draft",
+            "company_id": picking.company_id.id,
+            "picking_type_id": self._get_picking_type().id,
+            "product_uom_qty": self.quantity,
+            "product_uom": self.repair_order_id.move_id.product_uom.id,
+            "quantity": 0,
+            "repair_id": self.repair_order_id.id,
```

Comment:
```suggestion
            "related_repair_id": self.repair_order_id.id,
```

This has changed to `related_repair_id` according to this commit: https://github.com/OCA/repair/pull/77/commits/814fec812a5fbbb6d526cc0664710a47ed37a2f9